content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -314,18 +315,17 @@ Server:  Message Received
 ```
 4.The server receives and simultaneously shows;
 
-```bash
+```
 Connected
 Client: Hey there 
 Client: am doing this for the first time
 Client: BYE
 Closing connection
 ```
  Important note
-Sending "BYE" closes the connection between the Client and the Server.
 Port number may fail to work you can always change it and use a different one from those commercially owned.
 
 ### Conclusion 
 
-In this article, we have been able to get an insight of sockets and TCP/IP protocol. We have understood what socket programming entails and its relation to Java. We have also seen a clear picture of client/server relationship, how server accepts information from client and sends it back.
-I urge the reader to use the knowledge learned above to write programs on how server accepts the clients' request in java programming language.
\ No newline at end of file
+In this article,we got an insight into sockets and TCP/IP protocol. We went through what socket programming entails and its relation to Java. We have also seen a clear picture of client/server relationship, including the flow of data between the two.
```

Comment:
Change:

>In this article,we got an insight into sockets and TCP/IP protocol. We went through what socket programming entails and its relation to Java. We have also seen a clear picture of client/server relationship, including the flow of data between the two.
I urge the reader to use the knowledge learned above to write networking programs on how server accepts the client's request in java programming language.

To:

> In this article, we got an insight into sockets and TCP/IP protocol. We went through what socket programming entails and its relation to Java. We have also seen a clear picture of client/server relationships, including the flow of data between the two.
> 
> I urge the reader to use the knowledge learned above to write networking programs in Java.
> 
Fixed the lack of spaces in certain places, pluralized a word where that was missing, improved readability. Note: **networking programs** already imply the program involves a server accepting client requests. So I left out the part where you said:
**on how server accepts the client's request**.